latency/latency.go

```diff
@@ -0,0 +1,104 @@
+package latency
```

Comment:
Is there any benefit of having this repo here instead of putting it inside `network`?

Reply:
We will move latencies.json, which will be embedded, to this folder so I think it would be good to have all latencies related in same folder/package